spec/services/bulk_upload/sales/year2022/row_parser_spec.rb

```diff
@@ -521,4 +521,24 @@
       end
     end
   end
+
+  describe "inferences" do
+    context "when buyer not interviewed" do
+      let(:attributes) { valid_attributes.merge(field_6: "1", field_24: nil, field_30: nil, field_31: nil, field_32: nil, field_34: nil, field_36: nil, field_37: nil, field_39: nil, field_48: nil, field_49: nil) }
```

Comment:
Not that important, but a different test could be to check that even if sensible **non-nil** answers are given for these fields, they are still overridden correctly.

Reply:
I think actually we don't want to override those answers, but yes we could test that *doesn't* happen

Reply:
have done this :)